fixtures/canary/ms-cog-sci-fixed/swagger.json

```diff
@@ -0,0 +1,649 @@
+{
```

Comment:
These are specs taken from actual services. What was fixed?

Reply:
Theses specs are somehow invalid for go-swagger (docker, ms-cog, bitbucket).
I kept the original version so we can continue to measure our success against this yardstick.

However, for cases of legitimate spec invalidation by `swagger validate`, I produced a "fixed" version which is valid and allow us to check our full generation.

Spec "fixes" are as follows:
- docker: made OperationId unique
- ms-cog: duplicate property in allOf construct (which is valid JSON but refused by our validator, which is debatable, I don't know the expected behavior from Swagger 2.0 in this case)

We have other problems for valid canary specs.
These ones are not fixable at the spec level: there are simply excluded from the test run:
- bitbucket: wrong model generation
- kubernetes: use of options in mime type is not supported (produces duplicates in switch constructs) (kubernetes spec produces a valid client, though)